views/default/css/elements/misc/spinner.css

```diff
@@ -0,0 +1,34 @@
+/* ***************************************
+	FIXED POSITION AJAX INDICATOR
+*************************************** */
+
+.elgg-spinner {
+    position: fixed;
+    top: 0;
+    left: 50%;
```

Comment:
Add negative left margin with half of the width to keep actually centered. At width 51px, margin-left: -25.5px